.github/workflows/test-application.yaml

```diff
@@ -193,3 +193,104 @@ jobs:
             - name: Build admin ui
               run: npm run build
               working-directory: ${{ matrix.working-directory }}/assets/admin
+
+    php-windows:
+        name: "PHP ${{ matrix.php-version }} on Windows (mysql ${{ matrix.mysql-version }}, ${{ matrix.phpcr-transport }}, ${{ matrix.dependency-versions }})"
+        runs-on: windows-latest
+
+        env:
+            JACKRABBIT_VERSION: 2.20.2
+            APP_ENV: test
+            APP_SECRET: a448d1dfcaa563fce56c2fd9981f662b
+            MAILER_URL: null://localhost
+            SULU_ADMIN_EMAIL:
+            SYMFONY_DEPRECATIONS_HELPER: weak
+            PHPCR_TRANSPORT: jackrabbit
+            DATABASE_URL: mysql://root:root@127.0.0.1:3306/sulu_test?serverVersion=${{ matrix.mysql-version }}
+            DATABASE_CHARSET: utf8mb4
+            DATABASE_COLLATE: utf8mb4_unicode_ci
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - php-version: '7.4'
+                      mysql-version: '8.0'
+                      phpcr-transport: 'jackrabbit'
+                      dependency-versions: 'highest'
+                      php-extensions: 'ctype, iconv, intl, mysql, pdo_mysql, php_fileinfo, imagick'
+                      tools: 'composer'
+
+        steps:
+            - name: Checkout project
+              uses: actions/checkout@v2
+
+            - name: Install and configure Node.js
+              uses: actions/setup-node@v2
+              with:
+                  node-version: '14'
+
+            - name: Install and configure PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php-version }}
+                  extensions: ${{ matrix.php-extensions }}
+                  tools: ${{ matrix.tools }}
+                  ini-values: 'memory_limit=-1'
+
+            - name: Install and configure MySQL
+              uses: shogo82148/actions-setup-mysql@v1.0.1
+              with:
+                  mysql-version: ${{ matrix.mysql-version }}
+                  root-password: 'root'
+                  my-cnf: |
+                      [mysqld]
+                      default-authentication-plugin=mysql_native_password
+
+            - name: Cache jackrabbit runnable
```

Comment:
its a shame that installing jackrabbit is so complicated 🙈

Reply:
just copied it from sulu/sulu to make it work as fast as possible 🙈